### PR TITLE
Integrate formspark hook in newsletter and contact. update images and…

### PR DESCRIPTION
**Pull request by naura1835**
… create unit test for contact form

### CODE REVIEW

File: src/components/contact/Contact.js [hunk 1]
```diff
@@ -19,17 +20,26 @@ const inputs = [
...
   const [errors, setErrors] = useState({});
+  const [submit, submitting] = useFormspark({ formId: FORMSPARK_FORM_ID });
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault();
 
     const { newErrors, valid } = ValidateInput(formData);
 
     if (!valid) return setErrors(newErrors);
```
Comment (ebinabo) [inline src/components/contact/Contact.js:35]: does this line stop the form from attempting to submit anyway, or should we use a try/catch instead?

File: src/components/faq/Faq.js [hunk 2]
```diff
@@ -11,17 +11,18 @@ const Faq = () => {
       answer: "Yes, but refund is only valid for 72 hours",
     },
     {
-      question: "Can I pay using Cryptcurrency?",
+      question: "Can I pay using cryptcurrency?",
```
Comment (ebinabo) [inline src/components/faq/Faq.js]: this is spelt cryptocurrency

File: src/components/header/Header.js [hunk 3]
```diff
@@ -45,7 +46,10 @@ const Header = () => {
         className={`w-7 h-7 mr-4 cursor-pointer ${
           isMatched && "[&_path]:stroke-darkGrey"
         }`}
-        onClick={() => navigate("/cart")}
+        onClick={() => {
+          navigate("/cart");
```
Comment (ebinabo) [inline src/components/header/Header.js]: does setting state while navigating cause any rendering issues?

**Review by ebinabo — COMMENTED** [review]
Left a few comments there, they aren't blocking anything

File: src/components/header/Header.js (hunk 3 above)
Comment (naura1835) [reply src/components/header/Header.js]: Not that I noticed. Is there a better way to do it, cause without this, if the nav menu is opened on mobile and the cart icon is clicked, it navigates to the cart page but doesn't close the nav menu

File: src/components/contact/Contact.js (hunk 1 above)
Comment (naura1835) [reply src/components/contact/Contact.js:35]: Yes. will it work with a try/catch ?

**Pull request merged**